Resources/Prototypes/_Scp/Maps/scp_complex_compact.yml

```
@@ -36,7 +36,7 @@
             SeniorLogisticsOfficer: [ 1, 1 ]
             # Отдел внутренней безопасности
             # MentalControlOfficer: [ 1, 1 ]
-            InternalSecurityOperative: [ 1, 1 ]
+            #InternalSecurityOperative: [ 1, 1 ] # выводим до лучших времен
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)